frontend/src/components/Dashboard/CrossingListPage/CrossingListItem/CrossingListItem.js

```diff
@@ -31,13 +31,13 @@ class CrossingListItem extends React.Component {
     };
   }
 
-  newStatusUpdate(e) {
+  newStatusUpdate = (e) => {
     const updateData = {
       id: Math.round(Math.random() * -1000000),
       crossingId: this.props.crossing.id,
       statusId: this.state.selectedStatus,
-      reasonId: (this.state.selectedStatus !== statusConstants.OPEN ? this.state.selectedReason : null),
-      durationId: (this.state.selectedStatus === statusConstants.LONGTERM ? this.state.selectedDuration : null),
+      reasonId: this.state.selectedReason,
```

Comment:
@ifsimicoded just making sure this doesn't break anything:
One thing to check:
* Put crossing in open state, save
* Click Long-Term Closure, it should now be in a dirty state
* Set the reason and duration
* Click back on open, it should no longer be in dirty state

Reply:
Just checked out master and this branch, looks like the behavior is identical in both.